doc/quick_start/contributor-guide.rst

```diff
@@ -0,0 +1,277 @@
+.. _contributor-guide:
+
+
+Ostro |trade| OS Contributor Guide
+##################################
+
+As an open-source project, we welcome and encourage the community to submit patches directly to the project. 
+This document explains how to participate in project conversations, log bugs and enhancement requests, 
+and submit patches to the project so your patch will be accepted quickly in the codebase. At a high-level, 
+we use the GitHub Pull Request workflow and follow the `Open Embedded guidelines for commit messages`_, 
+as explained in our `Code Review Process Guidelines`_ section below.
+
+Participating in Ostro Project Conversations
+============================================
+
+As in many open source projects, we have online options for developers and community members to converse about the Ostro Project, including
+an `Ostro-dev mailing list`_, IRC channel, `ostroproject GitHub repo`_, bug and issue tracking, `Ostro OS Release Notes`_, and more.  
```

Comment:
Can we make "bug and issue tracking" a link to https://ostroproject.org/jira ? Same for IRC? Then each item in the list would be a link, which is convenient and consistent.